docs/QUnit/module.md

```diff
@@ -268,3 +290,103 @@ QUnit.module( "Database connection", {
   }
 } );
 ```
+
+---
+
+Only run a subset of tests:
+
+```js
+QUnit.module( "Tank", function() {
```

Comment:
Sorry about that.

I'll update it per suggestion.

Thanks for pointing that out!